tests/cli/workflow/taxprofiler/conftest.py

```diff
@@ -0,0 +1,21 @@
+"""Fixtures for cli workflow taxprofiler tests"""
+
+import datetime as dt
+import gzip
+from pathlib import Path
+from typing import List
+
+import pytest
+from cg.constants import Pipeline
+from cg.constants.constants import FileFormat
+from cg.meta.workflow.taxprofiler import TaxprofilerAnalysisAPI
+from cg.models.cg_config import CGConfig
+
+
+@pytest.fixture(scope="function", name="taxprofiler_context")
+def fixture_taxprofiler_context(
+    cg_context: CGConfig,
+) -> CGConfig:
+    """context to use in cli"""
```

Comment:
```suggestion
    """Context to use in cli."""
```
to follow the convention start the descriptions with upper case and end with a dot